src/gulpfile.js

```diff
@@ -255,6 +255,25 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor'], function () {
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(gulp.src(paths.app + '/env/template-editor/' + process.env.NODE_ENV + '.js'), {
+    transform: function(filePath, file) {
+      return '<script>' + file.contents.toString() + '</script>';
+    }
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
+});
+
+gulp.task('template-editor', function () {
+  return gulp.src([
+      paths.app + '/template-editor/adapterService.js'
```

Comment:
Oh, we forgot it, but we should include revision number in this js, in another case we could cache unwanted things.

Maybe bundle all js files could help on that.

Reply:
Yes, the bundle approach will prevent the cache